fuuk/people/admin/forms.py

```diff
@@ -49,3 +50,12 @@ class Meta:
             'type', 'year', 'title', 'presenter',  # talk data
             'publication', 'volume', 'page_from', 'page_to', 'editors', 'place'  # abstract collection data
         )
+
+
+class GrantCollaboratorInlineFormSet(forms.BaseInlineFormSet):
+    def clean(self):
+        super(GrantCollaboratorInlineFormSet, self).clean()
+        investigator_human = self.instance.investigator_human
+        for form in self.forms:
+            if form.cleaned_data and investigator_human and investigator_human == form.cleaned_data['human']:
```

Comment:
I don't understand the point. If there is no collaborator, the `form.cleaned_data` is empty and therefore `form.cleaned_data['human']` will raise `KeyError`, which I don't want. And when there is no collaborator, it is not necessary to test for the existence of `investigator.human`.